projects/media-viewer/src/lib/viewers/image-viewer/image-viewer.component.html

```diff
@@ -6,7 +6,7 @@
   >
     <img #img *ngIf="url" [ngClass]="'rot' + rotation" data-hook="dm.viewer.img" [src]="url" [style.width]="img.naturalWidth * zoom + 'px'" (error)="onLoadError()"/>
     <mv-annotation-set [annotationSet]="annotationSet"
-                       [drawMode]="drawMode"
+                       [drawMode]="toolbarEventsService.drawMode"
```

Comment:
could we extract this property within the component rather than the view. it seems like the view doesn't need to know about the service, just the draw mode